ynput/OpenPype
# Blender: output node and EXR

## Changelog Description
Output node now works correctly for Multilayer EXR and keeps existing links. The output now is handled entirely by the compositor node tree.

## Additional info
Output node in the compositing node tree wasn't working correctly for Multilayer EXR. Now existing links are recreated when creating an instance. Moreover, now the output is handled only by the compositor tree. Finally, when creating a render instance, a new version of the workfile is created, instead of overwriting the current one.

## Testing notes:
1. Set the output to be multilayer EXR.
2. Create a new render instance.
3. Check if the output node in the compositor node tree has all the necessary links.
4. Try deleting the instance (but not the node in the tree), and recreate the instance. The node should have the same links as before.
5. Publish the render and check that the layers are correct.
6. Change to PNG or any other format, and try publish it again, and check that the render is ok, expecially the beauty render.
